app/views/ThreadMessagesView/Item.stories.js

```diff
@@ -38,6 +38,7 @@ const BaseItem = ({ item, ...props }) => (
 		}}
 		onPress={() => alert('pressed')}
 		{...props}
+		thread
```

Comment:
You changed the component a lot to accept discussions, so you need to write stories for that